build.gradle

```diff
@@ -186,6 +177,103 @@ configure(subprojects.findAll { subprojectNamesReleased.contains(it.name) }) {
     }
 }
 
+configure(subprojects.findAll { subprojectNamesReleasedInMavenCentral.contains(it.name) }) {
+    apply plugin: "signing"
+
+    jar {
+        from rootProject.file("LICENSE")
+    }
+
+    javadoc {
+        options {
+            overview = "src/main/html/overview.html"
+            links "https://docs.oracle.com/javase/8/docs/api/"
+        }
+    }
+
+    publishing {
+        publications {
+            maven(MavenPublication) {
+                groupId = "${project.group}"
+                artifactId = "${project.name}"
+
+                from components.java
+                // javadocJar and sourcesJar are added by java.withJavadocJar() and java.withSourcesJar() above.
+                // See: https://docs.gradle.org/current/javadoc/org/gradle/api/plugins/JavaPluginExtension.html
+
+                pom {  // https://central.sonatype.org/pages/requirements.html
+                    name = "${project.name}"
+                    afterEvaluate { project ->
+                        // "description" is declared in subproject's build.gradle. It needs to be configured after evaluation.
+                        description = "${project.description}"
+                    }
+                    url = "https://www.embulk.org/"
+
+                    licenses {
+                        license {
+                            // http://central.sonatype.org/pages/requirements.html#license-information
+                            name = "The Apache License, Version 2.0"
+                            url = "https://www.apache.org/licenses/LICENSE-2.0.txt"
+                        }
+                    }
+
+                    developers {
+                        developer {
+                            name = "Sadayuki Furuhashi"
+                            email = "frsyuki@gmail.com"
+                        }
+                        developer {
+                            name = "Muga Nishizawa"
+                            email = "muga.nishizawa@gmail.com"
+                        }
+                        developer {
+                            name = "Satoshi Akama"
+                            email = "satoshiakama@gmail.com"
+                        }
+                        developer {
+                            name = "Dai MIKURUBE"
+                            email = "dmikurube@treasure-data.com"
+                        }
+                        developer {
+                            name = "Shinichi Ishimura"
+                            email = "shiketaudonko41@gmail.com"
```

Comment:
@kamatama41 Do you mind if we include you, with your e-mail address in Embulk's Git log, as one of the `<developers>` in POM of Embulk Maven artifacts? It is to be uploaded in Maven Central. (We haven't have `<developers>` in Bintray, but it is mandatory in Maven Central.)

Reply:
@dmikurube 
No problem, I'm so proud of it 😃 
(I haven't checked recent Embulk topics for a long time, though)

Reply:
Thanks! Yeah, we're happy to have you in the list. :)